.github/workflows/fidelity-tests.yml

```diff
@@ -35,10 +35,10 @@ jobs:
     - name: Bootstrap packages
       run: npm run bootstrap
 
-    - name: Build packages
-      run: npm run build
+    - name: Build packages # may be just build render fidelity packages?
+      run: ./node_modules/.bin/lerna run --scope @google/model-viewer-render-fidelity-tools build --stream
```

Comment:
maybe just build the fidelity test tool pacakge? It saves some time

Reply:
Maybe; what version of model-viewer will be used for the fidelity tools in this case? We needs to ensure it is building the master version of model-viewer and not using the last release on npm.